ocs_ci/templates/workloads/jenkins/jenkins_template

```diff
@@ -0,0 +1,246 @@
+apiVersion: template.openshift.io/v1
+kind: Template
```

Comment:
We are combining almost 6 different kind of yamls in a single yaml. 
How are you handling the clean up?
Do you think it would be better to have separate yamls for each kind?